app/index.js

```diff
@@ -371,6 +407,34 @@ function onClickHangup() {
   stopLocalMedia();
 }
 
+function onClickAVCtrl(isMuteOrPaused, isAudio) {
+  var logMsg = 'Audio is '
+    +(!isMuteOrPaused?'muted':'unmuted')+' for ';
+  if (isAudio === false ) {
+    logMsg = 'Video is '
+      +(!isMuteOrPaused?'paused':'resumed')+' for ';
+  }
+  var mediaTracks = isAudio ?
+    window.localStream.getAudioTracks() : window.localStream.getVideoTracks();
+  if (mediaTracks.length === 0) {
+    console.warn('No local ',isAudio ? 'audio':'video','available.');
+    return;
+  }
+  for (var i = 0; i < mediaTracks.length; i+=1) {
+    mediaTracks[i].enabled = isMuteOrPaused;
+  }
+  const pcs = lib.getPCObjects();
+  for(const key in pcs) {
+    if (pcs.hasOwnProperty(key) && pcs[key]) {
+      console.log(logMsg + pcs[key].userId);
+      handleApplicationLogs(
```

Comment:
Ths sends only the log event, but the fabric event, I think we stilll have to integrate sendFabricevent to report the mute/unmute pause/resume 
https://www.callstats.io/api/#enumeration-of-fabricevent

Reply:
Yes, it was my mistake. Added missing sendFabricEvent. Thanks for helping me with it.